### PR TITLE
Hartono

### PR DESCRIPTION
**Pull request by louishartono**
Louis' rst file

### CODE REVIEW

File: source/people/louishartono.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+| Madison, WI 53711
+| lehartono@wisc.edu
+| Phone (608) 332-6704
+
+--------------------
+
+I am a sophomore majoring in Industrial Engineering and Economics with mathematical emphasis
+
+Current Projects
+--------------------
```
Comment (bam241) [inline source/people/louishartono.rst]:
```suggestion
----------------
```

File: source/people/louishartono.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Computational Tool Development for Nuclear Fuel Cycle Analysis
+
+Education
+--------------------
```
Comment (bam241) [inline source/people/louishartono.rst]: this one as well

File: source/people/louishartono.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
+Louis Evantio Hartono
+=====================
+
+
+Contact Information
+-------------------
+
+| 1423 Monroe St Apt #302
```
Comment (gonuke) [inline source/people/louishartono.rst]: I don’t think you need to give your home address here.   Either use your office number or don’t use any address.

File: source/people/main.rst [hunk 4]
```diff
@@ -110,11 +110,11 @@ Undergraduate Students & Interns
 ---------------------------------
 
 
-+------------------------------------+------------------------------------+-------------------------------------+
-|                                    |                                    |                                     |
-|                                    |                                    |                                     |
-| :doc:`Piper Lincoln <plincoln>`    | :doc:`Matthew Nyberg <mnyberg>`    | :doc:`Brad Laufenberg <laufenberg>` |
-+------------------------------------+------------------------------------+-------------------------------------+
++------------------------------------+------------------------------------+-------------------------------------+--------------------------------------
```
Comment (gonuke) [inline source/people/main.rst]: Need to put a `+` at the end of this line
```suggestion
+------------------------------------+------------------------------------+-------------------------------------+-------------------------------------+
```

**Comment by gonuke** [comment]
Also, there have now been other changes to `main.rst`, so you might want to get some tips on how to "`rebase`" your branch from @bam241

**Comment by gonuke** [comment]
Welcome @louishartono

**Pull request merged**